x-pack/plugins/cases/public/client/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './api';
```

Comment:
I am not a fan of re-exports like this, specially barrel re-exports that had created memory leaks in the past specially on jest tests. Do we need it here?

I'd say we can simply import these directly whenever we use them and this additional re-export is unnecessary.

The re-export also messes with "go to defintion" tools because you end up in this file instead of the definition you actually want to see.

Reply:
Ok!